js/store.js

```diff
@@ -45,17 +45,17 @@ const store = {
     return await browser.storage.local.set({ websites });
   },
 
-  outputWebsite(hostname, website, firstPartyHostname) {
+  outputWebsite(hostname, website) {
     const output = {
       hostname: hostname,
       favicon: website.faviconUrl || '',
-      firstPartyHostname: firstPartyHostname || false,
+      firstPartyHostnames: website.firstPartyHostnames || false,
       firstParty: false
     };
```

Comment:
@jonathanKingston can we add `thirdParties: []` here and leave out the below `else` statement? In that way, the `output` object's signature would be complete.